bpfman/src/bin/cli/args.rs

```diff
@@ -307,9 +312,241 @@ pub(crate) struct GetArgs {
+            GoArch::Ppc64 => format!("BC_PPC64_EB={}", bc.display()),
+            GoArch::Ppc64le => format!("BC_PPC64LE_EL={}", bc.display()),
+            GoArch::Riscv64 => format!("BC_RISCV64_EL={}", bc.display()),
+            GoArch::S390x => format!("BC_S390X_EB={}", bc.display()),
+        }
+    }
+
+    pub(crate) fn from_cilium_ebpf_file_str(s: &str) -> Result<Self, std::io::Error> {
+        if s.contains("bpf_x86_bpfel.o") {
+            Ok(GoArch::Amd64)
+        } else if s.contains("bpf_arm_bpfel.o") {
+            Ok(GoArch::Arm)
+        } else if s.contains("bpf_arm64_bpfel.o") {
+            Ok(GoArch::Arm64)
+        } else if s.contains("bpf_loongarch_bpfel.o") {
+            Ok(GoArch::Loong64)
+        } else if s.contains("bpf_mips_bpfeb.o") {
+            Ok(GoArch::Mips)
+        } else if s.contains("bpf_powerpc_bpfeb.o") {
+            Ok(GoArch::Ppc64)
+        } else if s.contains("bpf_powerpc_bpfel.o") {
+            Ok(GoArch::Ppc64le)
+        } else if s.contains("bpf_riscv_bpfel.o") {
+            Ok(GoArch::Riscv64)
+        } else if s.contains("bpf_s390_bpfeb.o") {
+            Ok(GoArch::S390x)
+        } else {
+            Err(std::io::Error::new(ErrorKind::InvalidInput, "not a valid cilium/ebpf bytecode filename, please refer to https://github.com/cilium/ebpf/blob/main/cmd/bpf2go/gen/target.go#L14"))
+        }
+    }
+}
+
+#[derive(Args, Debug)]
+#[command(disable_version_flag = true)]
+pub(crate) struct BuildBytecodeArgs {
+    #[clap(flatten)]
+    pub(crate) bytecode_file: BytecodeFile,
+
+    /// Required: Name and optionally a tag in the name:tag format.
+    /// Example: --tag quay.io/bpfman-bytecode/xdp_pass:latest
+    #[clap(short, long, verbatim_doc_comment)]
+    pub(crate) tag: String,
+
+    /// Required: Dockerfile to use for building the image.
+    /// Example: --file Containerfile.bytecode
+    #[clap(short = 'f', long, verbatim_doc_comment)]
+    pub(crate) container_file: PathBuf,
+
+    /// Optional: Container runtime to use, works with docker or podman, defaults to docker
+    /// Example: --runtime podman
+    #[clap(short, long, verbatim_doc_comment)]
+    pub(crate) runtime: Option<String>,
+}
+
+#[derive(Args, Debug)]
+#[clap(group(
+    ArgGroup::new("bytecodefile")
+        .multiple(false)
+        .conflicts_with("multi-arch")
+        .args(&["bytecode", "cilium_ebpf_project"]),
+))]
+#[clap(group(
+    ArgGroup::new("multi-arch")
+        .multiple(true)
+        .args(&["bc_386_el", "bc_amd64_el", "bc_arm_el", "bc_arm64_el", "bc_loong64_el", "bc_mips_eb", "bc_mipsle_el", "bc_mips64_eb", "bc_mips64le_el", "bc_ppc64_eb", "bc_ppc64le_el", "bc_riscv64_el", "bc_s390x_eb"]),
+))]
+#[command(disable_version_flag = true)]
+#[group(required = true)]
+pub(crate) struct BytecodeFile {
+    /// Optional: bytecode file to use for building the image assuming host architecture.
+    /// Example: -b ./bpf_x86_bpfel.o
```

Comment:
To be consistent with the other examples:
```suggestion
    /// Example: -b ./examples/go-xdp-counter/bpf_x86_bpfel.o
```